src/main/resources/jenkins/security/plugins/ldap/LDAPConfiguration/help-ignoreIfUnavailable.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+    If checked, this option indicates that authentication and user/group membership lookups
+    will be attempted against subsequent LDAP domains even if this domain is unavailable.
```

Comment:
_perhaps just my english level_ 
this sentence seems to say that if the domain is available and respond correctly the subsequent domains will *still* be lookup.
From my understanding of the initial problem, the goal is to let administrators say that a domain is somewhat optional, if it fails, we try the next one. If it succeeds, we use that information.

Reply:
Good point, I will try reword this to make it very clear that it only happens in the event of failure.